.github/workflows/ci.yml

```diff
@@ -16,8 +16,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rust: ["stable", "beta", "nightly", "1.65"] # MSRV
+        rust: ["stable", "beta", "nightly"]
```

Comment:
are we updating our MSRV policy?

Reply:
No, just not testing `--all-features` because this feature requires a higher MSRV